tasks/main.yml

```diff
@@ -1,5 +1,12 @@
 ---
 
+- name: ensure firewalld is installed
+  become: true
```

Comment:
Since the whole role requires superuser privileges, this `become` is superfluous.

Reply:
Ah right, I removed it! Is there a special reason for not allowing the latest version of a package?